.github/workflows/test_spack.yml

```
@@ -30,9 +30,9 @@ jobs:
 
     - name: Prepare Spack configuration
       run: |
+        apt update && apt install unzip
```

Just to make sure that `apt` does not hang on any confirmation etc.:
```suggestion
        apt update -y -q && apt install -y -q unzip
```